packages/admin/cms-admin/src/form/file/FileUploadField.tsx

```diff
@@ -1,19 +1,29 @@
 import { Field, type FieldProps } from "@comet/admin";
 
-import { FinalFormFileUpload } from "./FinalFormFileUpload";
+import { FinalFormFileUpload, type FinalFormFileUploadProps } from "./FinalFormFileUpload";
 import { type GQLFinalFormFileUploadDownloadableFragment, type GQLFinalFormFileUploadFragment } from "./FinalFormFileUpload.generated";
 
-type SingleFileUploadProps = FieldProps<GQLFinalFormFileUploadFragment | GQLFinalFormFileUploadDownloadableFragment, HTMLInputElement> & {
-    multiple?: false;
-    maxFiles?: 1;
-};
+type SingleFileUploadProps<Multiple extends boolean | undefined> = FieldProps<
+    GQLFinalFormFileUploadFragment | GQLFinalFormFileUploadDownloadableFragment,
+    HTMLInputElement
+> &
+    Partial<FinalFormFileUploadProps<Multiple>> & {
```

Comment:
Would it be possible to set the multiple generic to `false` here?

Reply:
indeed - simplified it.